app/controllers/posts_controller.rb

```diff
@@ -8,9 +8,19 @@ def index
     def show
         @post = Post.find(params[:id])
 
-
     end 
 
+    def destroy
+        @post = Post.find(params[:id])
+            if @post.destroy
+                flash[:notice] =  "'"+ @post.title + "'" +" successfully deleted."  
```

Comment:
So this is good and it works as you'd expect. However :) we prefer to use a technique called string interpolation.  It's a little tidier and easier to read.

Have a look at http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html

And try to convert this line to this style